app/src/main/java/org/simple/clinic/instantsearch/InstantSearchEffect.kt

```diff
@@ -21,3 +21,5 @@ data class ValidateSearchQuery(val searchQuery: String) : InstantSearchEffect()
 data class OpenPatientSummary(val patientId: UUID) : InstantSearchEffect()
 
 data class OpenLinkIdWithPatientScreen(val patientId: UUID, val identifier: Identifier) : InstantSearchEffect()
+
+data class OpenBpPassportSheet(val identifier: Identifier) : InstantSearchEffect()
```

Comment:
Hmm, I was following the naming conventions of `OpenPatientSummary` to indicate which screen/sheet we will be opening. But `CheckNewRegistrationSheet` name sounds good as well, I can change it if you want.

Reply:
It's not about whether I prefer this name, it's about what makes more sense from a business use case perspective.

Reply:
In any case, we can discuss this later. It's not critical to this PR.

Reply:
Ah okay, my only issue with `CheckNewRegistrationSheet` is its kind of misleading since we are not actually checking or validating anything. We are just opening a sheet, where the user has to select one of the 2 buttons to either create a patient or assign it to a patient. So `OpenBpPassportSheet` makes more sense in my opinion. WDYT?

Reply:
Ok, I don't think `OpenBpPassport` is the right name for this feature if that's the scenario but `CheckNewRegistration` is also not the right one. Let's table this for now and figure out a better name later.